shared/js/content-scripts/click-to-load.js

```diff
@@ -1,4 +1,7 @@
 (function clickToLoad () {
+    function sendMessage (messageType, options, callback) {
+        chrome.runtime.sendMessage({ messageType, options }, callback)
```

Comment:
no browser / promises in here? I guess we don't want to inject the polyfil as a content script, right?

Reply:
Not yet at least, I'm just changing the `messageType` shape so it's easier to process in the onMessage handler.